docs/core/plugins/fts_solr.md

```diff
@@ -231,8 +231,19 @@ it are:
 - Configure Dovecot's userdb lookup to return a different host for
   [[setting,fts_solr_url]] setting using [[link,userdb_extra_fields]].
 
-  - LDAP: `user_attrs = ..., solrHost=fts_solr_url=https://%$:8983/solr/dovecot/`
   - MySQL: `query = SELECT concat('url=https://', solr_host, ':8983/solr/dovecot/') AS fts_solr, ...`
```

Comment:
while you're changing things here, fix this also `... concat('https://', solr_host, ':8983/solr/dovecot/') AS fts_solr_url, ...`